doc/fr/upgrade/from_packages_1810.rst

```diff
@@ -53,7 +53,22 @@ Mettez à jour l'ensemble des composants : ::
 Actions complémentaires
 =======================
 
-Centreon 19.04 utilise un nouveau paquet pour le serveur Web Apache.
+Mise à jour de la version de PHP
+--------------------------------
+
+Centreon 19.10 utilise un nouveau paquet PHP
+
+Réalisez les actions suivantes : ::
+
+    # systemctl disable rh-php71-php-fpm
+    # systemctl stop rh-php71-php-fpm
+    # systemctl stop rh-php72-php-fpm
+    # systemctl enable rh-php72-php-fpm
```

Comment:
enable before start ?

Reply:
Order doesn't matter, _start_ and _enable_ do different things.